edbdeploy/project.py

```diff
@@ -638,6 +638,19 @@ def deploy(self, no_install_collection):
         # Display inventory informations
         self.display_inventory(inventory_data)
 
+    def display_details(self):
+        try:
+            states = self.load_states()
+        except Exception as e:
+            states={}
+        status = states.get('ansible', 'UNKNOWN')
+        if status == 'DEPLOYED':
+            inventory_data = None
+            ansible = AnsibleCli(self.project_path)
+            with AM("Extracting data from the inventory file"):
+                inventory_data = ansible.list_inventory(self.ansible_inventory)
+            self.display_inventory(inventory_data)
+
```

Comment:
I think we should handle the case `status != 'DEPLOYED'` and display a warning message in this case.

Reply:
Done. Can you please review the new changes?